pkg/sql/execinfrapb/processors_sql.proto

```diff
@@ -513,18 +513,15 @@ message InvertedJoinerSpec {
   // inverted column, and the the remaining columns are the primary key.
   optional uint32 index_idx = 2 [(gogoproto.nullable) = false];
 
-  // Index of the column in the input stream that is to be joined with
-  // the inverted index.
-  optional uint32 lookup_column = 3 [(gogoproto.nullable) = false];
```

Comment:
Mark this id as reserved so it's not reused.

Reply:
Good call - Done.